chain/client/src/sync/state.rs

```diff
@@ -136,11 +134,11 @@ pub struct StateSync {
 
     /// When the "sync block" was requested.
     /// The "sync block" is the last block of the previous epoch, i.e. `prev_hash` of the `sync_hash` block.
-    last_time_block_requested: Option<DateTime<Utc>>,
+    pub last_time_block_requested: Option<DateTime<Utc>>,
 
     /// Timeout (set in config - by default to 60 seconds) is used to figure out how long we should wait
     /// for the answer from the other node before giving up.
-    timeout: Duration,
+    pub timeout: Duration,
```

Comment:
could avoid making this one pub by just using `self.config.state_sync_timeout` in the client actor